src/Tools/babylon.database.ts

```diff
@@ -11,7 +11,7 @@ module BABYLON {
         private isSupported: boolean;
 
         // Handling various flavors of prefixed version of IndexedDB
-        private idbFactory = <IDBFactory> (window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB);
+        private idbFactory = <IDBFactory> (window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || (window as any).msIndexedDB);

```

Comment:
They have probably removed msIndexedDB from their d.ts, we have to add it to our mixin indeed rather than casting it as any.